ansible/_list-images.yaml

```
@@ -0,0 +1,4 @@
+---
+  - hosts: localhost
+    roles:
+      - role: images
```

I think we want this to be its own role instead of `images`
```suggestion
    tasks:
    - name: list images
      shell: "echo {{ item }} >> ../images.out"
      loop: "{{ control_plane_images + extra_images + aws_images }}"
```

---

This is also not in conflict with https://github.com/mesosphere/konvoy-image-builder/pull/161/files, awesome!

---

Tried the above out and it fails to create the images. not sure how to fix it atm but havent dug in deep enough.